dataland-frontend/tests/e2e/fixtures/FixtureUtils.ts

```diff
@@ -49,12 +49,30 @@ export function generateArray<T>(
 }
 
 /**
- * Picks a random number of distinct values from an array and returns them
- * @param inputArray is the array containing all available values to choose from
- * @returns an array with a random number of distinct values chosen from the inputArray
+ * Picks one element of a given array and returns it
+ * @param inputArray the array containing all available values to choose from
+ * @returns a single element of the given array
  */
-export function getRandomNumberOfDistinctElementsFromArray<T>(inputArray: T[]): T[] {
-  const arrayOfAvailableDistinctValues = [...Array.from(new Set(inputArray))];
-  const numElementsForResultArray = Math.floor(Math.random() * arrayOfAvailableDistinctValues.length) + 1;
-  return faker.helpers.uniqueArray(arrayOfAvailableDistinctValues, numElementsForResultArray);
+export function pickOneElement<T>(inputArray: T[]): T {
+  return faker.helpers.arrayElement(inputArray);
+}
+
+/**
+ * Picks one element of a given array and returns it
+ * @param inputArray the array containing all available values to choose from
+ * @returns a single element of the given array
```

Comment:
The description is inaccurate.
It doesn't return the single element. It returns a list, which either contains the one element, or no elements.

Reply:
yeah, copy paste :)